imxrt1062-rt/src/cache.rs

```diff
@@ -0,0 +1,88 @@
+//Cache configuration
+
+use core::ptr;
+
+static mut SCB_CCR: *mut u32 = 0xE000ED14 as *mut u32;
+static mut SCB_MPU_CTRL: *mut u32 = 0xE000ED94 as *mut u32;
+static mut SCB_MPU_RBAR: *mut u32 = 0xE000ED9C as *mut u32;
+static mut SCB_MPU_RASR: *mut u32 = 0xE000EDA0 as *mut u32;
+static mut SCB_CACHE_ICIALLU: *mut u32 = 0xE000EF50 as *mut u32;
```

Comment:
Consider making these `const`s instead of `static`s. [`const` pointers work the same](https://github.com/mciantyre/teensy4-rs/blob/master/imxrt1062-rt/src/fpu.rs#L13) and won't take up space in the output.

We can check this by inspecting the symbol output from an example. The commands below builds the LED example (if prompted, use CTRL+C to abort the deployment to the Teensy 4), then finds symbols matching `SCB_` in the output. We see the five `statics` in the `.data` section.
```
$ ./hardware-test.sh led
$ cat out/led.sym | grep SCB_

20000108 l     O .data	00000004 .hidden imxrt1062_rt::cache::SCB_MPU_CTRL
20000110 l     O .data	00000004 .hidden imxrt1062_rt::cache::SCB_MPU_RASR
2000010c l     O .data	00000004 .hidden imxrt1062_rt::cache::SCB_MPU_RBAR
20000114 l     O .data	00000004 .hidden imxrt1062_rt::cache::SCB_CACHE_ICIALLU
20000104 l     O .data	00000004 .hidden imxrt1062_rt::cache::SCB_CCR
```

Once these are `const`s, we won't see them in the output. We save 20 bytes and make it easier to scan the output for other statics 😝